### PR TITLE
modem_manager: add T99W373 need to restore defult configuration of autosuspend_delay_ms

### PR DESCRIPTION
**Pull request by Fanice-luo**
autosuspend_delay_ms

1:After mbim_qdu fw update successfully, Some devices need restore defult configuration of autosuspend_delay_ms:2000ms, we define fu_mm_device_get_device_info to distinguish different devices.
2:Due to different MCP configurations, the time of FU_MM_DEVICE_REMOVE_DELAY_REPROBE need to increase 180000ms to 210000ms.
3:Need to synchronize MAX_WAIT_TIME_SECS 150s->240s for the time of FU_MM_DEVICE_REMOVE_DELAY_REPROBE.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/modem-manager/fu-mm-device.c [hunk 1]
```diff
@@ -1153,6 +1164,106 @@ fu_mm_device_writeln(const gchar *fn, const gchar *buf, GError **error)
...
+	    g_file_get_contents (subsystem_device_path, &subsystem_device, NULL, NULL))
+	{
+		vendor_id = (gchar *)g_strchomp (subsystem_vendor);
+		device_id = (gchar *)g_strchomp (subsystem_device);
+		
+		for (guint i = 0; i < max(sdx12_lens, sdx55_lens, sdx62_lens); i++)
+		{
+			if (i < sdx12_lens)
+			{
+				for (guint a = 0; a < sdx12_lens; a++)
```
Comment (hughsie) [inline plugins/modem-manager/fu-mm-device.c:1228]: did you run ./contrib/setup? the formatting is pretty broken.

File: plugins/modem-manager/fu-mm-device.h [hunk 2]
```diff
@@ -40,4 +40,14 @@ fu_mm_device_udev_new(FuContext *ctx, MMManager *manager, FuMmDevice *shadow_dev
 void
 fu_mm_device_udev_add_port(FuMmDevice *self, const gchar *subsystem, const gchar *path, gint ifnum);
 
+gboolean 
+fu_mm_device_write_suspend_ms(FuDevice *device,
+			      const gchar *buf,
+			      GError **error);
+
+guint max(guint x, guint y, guint z);
```
Comment (hughsie) [inline plugins/modem-manager/fu-mm-device.h:48]: there's already `MAX` you can use.

File: plugins/modem-manager/fu-mm-device.c (hunk 1 above)
Comment (Fanice-luo) [reply plugins/modem-manager/fu-mm-device.c:1228]: Hi @hughsie 
Do you mean I need to execute "sudo ./contrib/ci/fwupd_setup_helpers.py install-dependencies -o ubuntu --yes"?

Comment (superm1) [reply plugins/modem-manager/fu-mm-device.c:1228]: just run `./contrib/setup` and it sets everything up.  redo your commits and they get fixed while committing.

**Pull request closed without merging**